src/utils/location_codes.R

```diff
@@ -37,10 +37,13 @@ names_to_iso3 <- function(location_names) {
 #'
 #' @export
 ison_to_iso3 <- function(ison) {
+  # list of numerical iso codes used from data provider but not officially recognized
+  # 2 is used by ACLED for the areas of Akrotiri and Dhekelia (British overseas territories in Cyprus)
+  not_recognized_ison <- c(2)
   iso3 <- character(length(ison))
   iso3[ison == 0] <- "XKX"
-  iso3[ison != 0] <- countrycode$countrycode(
-    sourcevar = ison[ison != 0],
+  iso3[! ison %in% c(not_recognized_ison, 0)] <- countrycode$countrycode(
+    sourcevar = ison[! ison %in% c(not_recognized_ison, 0)],
     origin = "iso3n",
     destination = "iso3c"
   )
```

Comment:
this works! it looks like Seth assigned `XKX` to `ison == 0` but then disregards that country code when converting back to `iso3c`, but then he deals with `XKX` and other custom names in the end with

https://github.com/OCHA-DAP/hdx-signals/blob/37fd8ab78787c40c7ad93911a741be306e4e52f7/src/utils/location_codes.R#L13-L32